test-collateral/generate-policy/Cargo.toml

```diff
@@ -16,6 +16,7 @@ http = "=0.2.4"
 log = "=0.4.13"
 policy-utils = { path = "../../policy-utils", features = ["std"] }
 ring = { git = "https://github.com/veracruz-project/ring.git", branch = "veracruz", features = ["non_sgx"] }
+veracruz-utils = {path = "../../veracruz-utils", features = ["std", "tz"]}
```

Comment:
nit: can you move `veracruz-utils` down to retain alphabetical ordering of the imports?